flowzone.yml

```
@@ -109,11 +119,15 @@
     run: gh release delete --yes '${{ github.event.pull_request.head.ref }}' || true
     env:
       <<: *gitHubCliEnvironment
+      GH_TOKEN: ${{ steps.gh_token.outputs.token || secrets.FLOWZONE_TOKEN }}
```

This is repeated a lot, it could make sense to use anchors/aliases, eg
```suggestion
      <<:
        - *gitHubCliEnvironment
        - *ghTokenEnv
```
```yaml
- &ghTokenEnv
  GH_TOKEN: ${{ steps.gh_token.outputs.token || secrets.FLOWZONE_TOKEN }}

---

+1

---

or as I suggested above, keep this line in `*gitHubCliEnvironment`

---

@klutchell there is only place where `gitHubCliEnvironment` is used at the job/env level, so `steps` won't work there.